### ISSUE TITLE
SQL Server 2016 schemaInspector.relationExists(tableName) Error

### ISSUE THREAD

**Issue by Pinellus** [javers/javers#806]
Hi,
I've got a problem starting the javers app.

The first start (without javers tables) is executed without problems.

The second start goes wrong, SQL Exception because javers tries to recreate the schema for the table jv_commit (which already exists in the DB)

Going through the code, I found the error in the **org.javers.repository.sql.schema.JaversSchemaManager**

```
private void ensureTable(String tableName, Schema schema) {

        if (schemaInspector.relationExists(tableName)) {
            return;
        }
        logger.info("creating javers table {} ...", tableName);
        schemaManager.create(schema);

    }
```

The method **ensureTable** does not recognize that in the DB SQL Server the table **jv_commit** already exists and therefore launches again the CREATE TABLE.

This behavior is not present in version 5.0.0 of Javers (which I'm currently using to get around the problem).

Also I think that the problem is attributable to the **polyjdbc** library

The version of SQL Server I use is:
Microsoft SQL Server 2016 - 13.0.1708.0 (X64)
The JPA setting for the dialect is: 
`spring.jpa.hibernate.dialect=org.hibernate.dialect.SQLServer2012Dialect`

**Comment by bartoszwalacik** [comment]
Well, as you already know, developing on non-OSS databases is hard for us, as we don't have integration tests for it.

It would be nice to have at least local tests running on Docker and  Testcontainers.
Could you help us to make this test runnable?
https://github.com/javers/javers/blob/master/javers-persistence-sql/src/test/groovy/org/javers/repository/sql/integration/mssql/MsSqlIntegrationTest.groovy

**Comment by bartoszwalacik** [comment]
Integration test based on docker is configured, it was easier than I thought. Looks like MS SQL Server is friendly for testing on Docker (as opposite to Oracle).

**Comment by bartoszwalacik** [comment]
After applying the fix in polyJdbc, situation is much better, but still, when I run `MsSqlIntegrationTest`, 9 tests out of 104 fail. Further work is needed to make all tests green.

**Comment by bartoszwalacik** [comment]
main issue with the schemaInspector is fixed in 5.3.2
